tests/async/test_async_checkpassword.py

```diff
@@ -0,0 +1,33 @@
+from unittest import mock
+
+from asgiref.sync import sync_to_async
+from django.conf.global_settings import PASSWORD_HASHERS
+from django.contrib.auth.hashers import get_hasher
+from django.contrib.auth.models import User
+from django.test import TestCase, override_settings
+
+
+class AsyncPasswordUpgradeTestCase(TestCase):
+    @override_settings(PASSWORD_HASHERS=PASSWORD_HASHERS)
+    async def test_check_password_upgrade(self):
+        """
+        async version of tests.auth_tests.test_models.AbstractUserTestCase.test_check_password_upgrade.
+        """
+        user = await sync_to_async(User.objects.create_user)(username="user", password="foo")
+        initial_password = user.password
+        self.assertTrue(await user.acheck_password("foo"))
```

Comment:
Use `assertIs(..., True)` instead as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
OK, no problem.